scripts/visualize_learning_rate.py

```diff
@@ -50,7 +50,10 @@ def calculate_lr(lr_config, epochs=100, optimizer_config=None):
     lr_data = []
     for ep in range(epochs):
         optimizer.step()
-        scheduler.step(ep)
+        try:
```

Comment:
but the deprecation is just a warning right? or is it an error

Reply:
probably better if we use the version of pytorch to determine which one to call

Reply:
when I try using MultiStepLR and visualize it, it shows different behavior

Reply:
okay, I think it's fine

Reply:
This one when scheduler.step() called, the correct behavior 

![step_lr_with_burnin](https://user-images.githubusercontent.com/7122701/91544489-02ad8780-e94a-11ea-99cd-a604964b919f.jpg)

This one whem scheduler.step(epoch) called, wrong behavior

![Figure_2](https://user-images.githubusercontent.com/7122701/91544556-15c05780-e94a-11ea-8877-b6deff6beeb2.png)

Reply:
but to unify our base class with the current pytorch behavior, we could copy the original pytorch implementation from here https://github.com/pytorch/pytorch/blob/master/torch/optim/lr_scheduler.py#L24

Reply:
probably for another issue for this one